client/app/widgets/CarePlanContacts/__tests__/index.spec.jsx

```diff
@@ -17,105 +17,121 @@ const contacts = [
 ];
 
 const component = <CarePlanContacts contacts={contacts} />;
-const error = new Error('Error');
 
 describe('CarePlanContacts', () => {
   afterEach(() => {
     jest.restoreAllMocks();
   });
 
   it('renders the component', () => {
-    let wrapper;
-    expect(() => {
-      wrapper = render(component);
-    }).not.toThrow();
-    expect(wrapper).not.toBeNull();
+    render(component);
+
+    expect(screen.getByRole('heading')).toBeInTheDocument();
+    expect(screen.getByText('New Contact')).toBeInTheDocument();
+    expect(screen.getByText(contacts[0].name)).toBeInTheDocument();
+    expect(screen.getByText(contacts[1].name)).toBeInTheDocument();
+    expect(screen.getByText(contacts[1].phone)).toBeInTheDocument();
   });
 
   describe('when editing a contact', () => {
     it('opens a modal and submits the form successfully', async () => {
-      const axiosPatchSpy = jest.spyOn(axios, 'patch').mockImplementation(() => Promise.resolve({
+      const newPhoneNumber = '4160000000';
+      jest.spyOn(axios, 'patch').mockResolvedValue({
         data: {
           id: 1,
           name: 'Test1 Lastname',
-          phone: '4160000000',
+          phone: newPhoneNumber,
         },
-      }));
-      const wrapper = mount(component);
-      wrapper.find('a[aria-label="Edit"]').at(0).simulate('click');
-      expect(wrapper.text()).toContain('Edit Contact');
-      expect(
-        wrapper.find('input[aria-label="Name"]').props().defaultValue,
-      ).toEqual('Test1 Lastname');
-      expect(
-        wrapper.find('input[aria-label="Phone number"]').props().defaultValue,
-      ).toEqual(undefined);
-      wrapper.find('input[aria-label="Phone number"]').instance().value = '4160000000';
-      await act(async () => {
-        await wrapper.find('input[type="submit"]').simulate('click');
       });
-      axiosPatchSpy();
-      expect(wrapper.text()).toContain('4160000000');
+      const { container } = render(component);
+
+      const editLink = container.querySelector('a[aria-label="Edit"]');
+      expect(editLink).toBeInTheDocument();
```

Comment:
I think this can be removed since the assertions after `editLink` is clicked work.